sympy/core/power.py

```diff
@@ -124,6 +124,15 @@ class Pow(Expr):
     | (-oo)**oo    | nan     |                                               |
     | (-oo)**-oo   |         |                                               |
     +--------------+---------+-----------------------------------------------+
+    | oo**I        | nan     | oo**e could be thought of as the limit of     |
+    | oo**(1 + I)  | zoo     | x**e for real x tending to oo. If e is I,     |
+    | oo**(-1 + I) | 0       | then the limit does not exists and nan is     |
```

Comment:
Maybe you can write it more generally, eg. ``oo**(a + I*b)``. If a is greater than 0 then it is ``oo``. Something on these lines. ``oo**(1 + I)`` and ``oo**(-1 + I)`` are just examples.